src/main/java/com/solvd/movie/kafka/config/KafkaProducerConfig.java

```diff
@@ -0,0 +1,44 @@
+package com.solvd.movie.kafka.config;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.LongSerializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+import reactor.kafka.sender.KafkaSender;
+import reactor.kafka.sender.SenderOptions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+public class KafkaProducerConfig {
+
+    private static final String BOOTSTRAP_SERVERS = "localhost:9092";
+    private static final String TOPIC = "movies";
+
+    @Bean
+    public SenderOptions<String, Long> senderOptions() {
```

Comment:
what is the reason to use ```Long``` as a value?

Reply:
My practical sense of using kafka in this application is: if the movie is deleted, all reviews for this movie should be deleted too. That's why I am using Long type for value, because I send movie's id to consumer